package.json

```diff
@@ -9,7 +9,7 @@
     "docs": "npx jsdoc src/*.js src/**/*.js ./JSDOC.md -d docs",
```

Comment:
could you try running `npm run explore`? I'm seeing some failures there. Just a guess, but suspect the `--formPath` parameter is maybe being ignored?

Reply:
For me, it turns out the errors were happening because `build-assets.js` was just not handling the case where the `build` directory does not exist.  I added a bit of logic to handle that case and now `npm run explore` works without errors.  Let me know if the error still persists for you in the latest version!

Reply:
Builds fine (failed on node 20, i'll open separate issue).  Everything appears to work fine - but I see a console error when loading forms

```
Error rendering form:  Error: ["Error trying to parse XML instance \"contact-summary\". Cannot read properties of undefined (reading 'children')"]
```

Reply:
:sweat_smile: This error was a fun one to track down!  I finally discovered that the recreation steps were:

- Open Project Explorer
- Open `app` form
- Open `contact` form
- See error in console.log

It turns out the issue was just that the Project Explorer was not cancelling/unloading a form before switching to a different one.  In some cases this is not a problem since all of the data in context just gets overwritten by data for the new form.  However, app forms can have contact-summary data and if that is loaded in context and not cleared before switching to a contact form, it causes weird issues when trying to process the contact-summary (since we don't support the contact-summary for contact forms).  The fix here was just to add logic to clear out the current form in the Project Explorer before switching to a new one.